### ISSUE TITLE
Preferences: Disable/change settings for default code content in scripts and object events

### ISSUE THREAD

**Issue by dicksonlaw583** [YoYoGames/GameMaker-Bugs#7808]
### Is your feature request related to a problem?

Since the beginning of GMS 2, I have not been a fan of the default placeholder content in scripts and object events. Right now they are like this:

```
// Script assets have changed for v2.3.0 see
// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
```

```
/// @description Insert description here
// You can write your code in this editor
```

I don't believe that these placeholders help novices by trying to do the job of tutorials, in fact they quickly become a hindrance by adding needless bloat and backspacing busy work to an end user's workload. They are some of the first things I turn off or reconfigure on a new installation. In particular, we are more than 4 years away from the initial release of GMS 2.3.0, yet the "Script assets have changed for v2.3.0" placeholder comment is still being inserted into new scripts. In my opinion, the default placeholder has outlived its usefulness and should be disabled or drastically minimized.

### Describe the solution you'd like

Going forward, I would like to see these two settings **disabled** by default and give users of all levels an uncluttered, distraction-free slate to write on:

- Preferences > Script Editor > Add default script content
- Preferences > Object Editor > Add default event content

Even if it is retained, it should be changed to something minimalistic such as `///@func ` for scripts and `///@desc ` for object events.

**Comment by backYard321** [comment]
I agree! I even see a lot of experienced users still have this default content in their code because they a) don't assume that it can be disabled, b) don't want to take the time finding the setting to disable/change it, or c) have gotten used to the clutter. And seeing people's objects with many events populated with "Insert description here" really is an eyesore.

**Comment by stuckie** [comment]
Just to clarify this:
For *new* users, these will be blank for you to put whatever you like in.
For *existing* users, we won't touch what's already there... you can either reset the default to blank it, or blank it yourselves.

**Comment by YYBartT** [comment]
Updated the Object Editor Preferences and Script Editor Preferences manual pages. Also mentioned that you can update to the new default value (no content) on existing installations using **Restore defaults**.

**Comment by alicemoretti** [comment]
Verified implemented in IDE v2024.1100.0.654  Runtime v2024.1100.0.680. 

Thank you.
